README.md

```
@@ -33,9 +33,9 @@ By combining plugins you can use it for several purposes:
 
 ## Examples
 
-Get a more detailed look on [our demo website](https://axe312ger.github.io/sqip/).
+Check out the [interactive demo](https://sqip.vercel.app/) to compare all plugins and configurations side by side.
 
-[![](demo/example.jpg)](https://axe312ger.github.io/sqip/)
+[![](demo/example.jpg)](https://sqip.vercel.app/)
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)